libenv/sysinfo.c

```diff
@@ -1645,6 +1659,88 @@ static void OSClasses(EvalContext *ctx)
 
 /*********************************************************************************/
 
+#ifdef __APPLE__
+static void Apple_Version(EvalContext *ctx)
+{
+    FILE *pp = NULL;
+
+    if ((!FileCanOpen("/usr/bin/sw_vers", "r") || ((pp = cf_popen("/usr/bin/sw_vers", "r", true)) == NULL)))
+    {
+        Log(LOG_LEVEL_VERBOSE, "Could not find macOS system version information.");
```

Comment:
Since a path is embedded in C code I would like to know what it is trying in the verbose log:

```suggestion
        Log(LOG_LEVEL_VERBOSE, "Could not find and run /usr/bin/sw_vers to find macOS system version information.");
```

I wonder too if this shouldn't be an error.